fe/fe-core/src/test/java/com/starrocks/service/FrontendServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present StarRocks Inc.
+
+package com.starrocks.service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.thrift.TException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.starrocks.catalog.Column;
+import com.starrocks.catalog.Database;
+import com.starrocks.catalog.DistributionInfo;
+import com.starrocks.catalog.HashDistributionInfo;
+import com.starrocks.catalog.KeysType;
+import com.starrocks.catalog.OlapTable;
+import com.starrocks.catalog.PartitionInfo;
+import com.starrocks.catalog.RangePartitionInfo;
+import com.starrocks.catalog.TableProperty;
+import com.starrocks.catalog.Type;
+import com.starrocks.catalog.View;
+import com.starrocks.catalog.DistributionInfo.DistributionInfoType;
+import com.starrocks.common.util.PropertyAnalyzer;
+import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.thrift.TGetTablesMetaRequest;
+import com.starrocks.thrift.TGetTablesMetaResponse;
+
+
+import mockit.Expectations;
+import mockit.Mock;
+import mockit.MockUp;
+import mockit.Mocked;
+
+public class FrontendServiceImplTest {
+
+    @Mocked
+    ExecuteEnv exeEnv;
+
+    @Mocked
+    GlobalStateMgr globalStateMgr;
+
+    @Test
+    public void testGetTablesMeta() throws TException {
+
+        Database db = new Database(1, "test_db");
+        
+        List<Column> columns = new ArrayList<>();
+        columns.add(new Column("c1", Type.ARRAY_BOOLEAN));
+        columns.add(new Column("c2", Type.ARRAY_BOOLEAN));
+
+        // OlapTable
+        RangePartitionInfo partitionInfo = new RangePartitionInfo(columns);
+        HashDistributionInfo distributionInfo = new HashDistributionInfo(10, columns);
+        OlapTable table = new OlapTable(1, "test_table", columns, KeysType.PRIMARY_KEYS, partitionInfo, distributionInfo);
```

Comment:
You should test all kinds of tables, duplicate, aggregate, unique and primary key tables.

Reply:
done